flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceFailoverWithWatermarkExtractor.java

```diff
@@ -88,16 +88,11 @@ protected void assertRecords(Table table, List<Record> expectedRecords, Duration
     Awaitility.await("expected list of records should be produced")
         .atMost(timeout)
         .untilAsserted(
-            () -> {
-              SimpleDataUtil.equalsRecords(
```

Comment:
This does not do anything. The method is just an "equals" implementation, but the result is not used anywhere.
So I removed the call.